docs/source/write_first_app.rst

```diff
@@ -101,889 +97,485 @@ bringing up a new one.
-      discovery: {enabled: true, asLocalhost: true} // using asLocalhost as this gateway is using a fabric network deployed locally
-    });
+.. code-block:: TypeScript
 
-    // Build a network instance based on the channel where the smart contract is deployed
-    const network = await gateway.getNetwork(channelName);
+  const channelName = 'mychannel';
+  const chaincodeName = 'basic';
 
+  const network = gateway.getNetwork(channelName);
+  const contract = network.getContract(chaincodeName);
 
-    // Get the contract from the network.
-    const contract = network.getContract(chaincodeName);
+When a chaincode package includes multiple smart contracts, on the `getContract() <https://hyperledger.github.io/fabric-gateway/main/api/node/interfaces/Network.html#getContract>`__
+call you can specify both the name of the chaincode and a specific smart contract to target. For example:
 
-When a chaincode package includes multiple smart contracts, on the `getContract() API <https://hyperledger.github.io/fabric-sdk-node/release-2.2/module-fabric-network.Network.html#getContract>`__ you can specify both the name of the chaincode package and a specific smart contract to target. For example:
+.. code-block:: TypeScript
 
-.. code:: bash
+  const contract = network.getContract('chaincodeName', 'smartContractName');
 
-  const contract = await network.getContract('chaincodeName', 'smartContractName');
+Fourth, the application initializes the ledger with some sample assets
+----------------------------------------------------------------------
 
-Fourth, the application initializes the ledger with some sample data
---------------------------------------------------------------------
+Now that we are at the point where we are actually invoking smart contract transactions, let’s
+go through them in sequence.
 
-Now that we are at the point where we are actually having the sample application submit transactions, let’s
-go through them in sequence. The application code snippets and invoked chaincode snippets
-are provided for each called function, as well as the terminal output.
+Immediately after initial deployment of the chaincode, the ledger is empty. The application uses ``submitTransaction()``
+to invoke the chaincode ``InitLedger`` transaction function, which populates the ledger with some sample data. Behind
+the scenes, ``submitTransaction()`` will use the Gateway peer to:
 
-The submitTransaction() function is used to invoke the chaincode ``InitLedger`` function to populate the
-ledger with some sample data. Under the covers, the submitTransaction() function will use service discovery
-to find a set of required endorsing peers for the chaincode, invoke the chaincode
-on the required number of peers, gather the chaincode endorsed results from those peers,
-and finally submit the transaction to the ordering service.
+  1. Endorse the transaction proposal by successfully executing the transaction function on sufficient endorsing peers
```

Comment:
Okay so these steps we have in the peers page I think. Let's make sure they match / are in sync.

Reply:
These describe the steps from the application perspective. Simplified description to try to avoid confusion.